Mage.Tests/src/test/java/org/mage/test/cards/single/dis/AzoriusAethermageTest.java

```diff
@@ -11,8 +11,8 @@
      * Whenever you bounce a permanent (tokens included) you may pay {1}, if you do, draw a card
      */
     @Test
-    @Ignore
     public void testBouncedLand() {
+        /*
```

Comment:
The card seems to work fine when testing it manually. But we should figure out why this test does work and had to be commented out.

Can be done as a separate issue.

Reply:
The base being the main branch.  There are no conflicts at the moment.

Reply:
Okay. Merge it in and I'll open an issue for the Azorius Aethermage test